### PR TITLE
"disable_z()" for Delta PSU

### PR DESCRIPTION
**Pull request by MRprusa3d**

### CODE REVIEW

**Comment by F0x06** [comment]
Hi, i just received and installed my DELTA PSU, can you explain me what these changes are, and do i need to merge them into my firmware ?

Thanks !

**Comment by MRprusa3d** [comment]
Hello,

> ... do i need to merge them into my firmware ?
not now - it is 'under developement' still ...

File: Firmware/variants/1_75mm_MK3S-EINSy10a-E3Dv6full.h [hunk 1]
```diff
@@ -22,6 +22,9 @@
 #define STEEL_SHEET
 #define HAS_SECOND_SERIAL_PORT
 
+// PSU
+#define PSU_Delta                                 // uncomment if DeltaElectronics PSU installed
```
Comment (mkbel) [inline Firmware/variants/1_75mm_MK3S-EINSy10a-E3Dv6full.h:26]: Isn't it possible to use existing Z_AXIS_ALWAYS_ON macro for this purpose?

**Review by DRracer — APPROVED** [review]
Applicable for MK3S only as proposed

**Comment by maximlevitsky** [comment]
What about us that have old silver PSU and updated to MK3S using the kit?

**Comment by GurliGebis** [comment]
Most likely a PSU upgrade kit coming out soon, with a coupon for free shipping, so people only have to pay for it, to be able to use the newer firmwares...
Or maybe they will just fix the code to work with both power supplies 😉

**Comment by DRracer** [comment]
This PR shall switch the z-motors into silent mode when the printer is idle. It shall help the Delta PSU reduce its internal noise. It shall help the T-Led silver PSU too, we'll test it in the upcoming few days again to avoid any regressions.

**Pull request merged**